azure-pipelines/end-to-end-tests-dir/bundles.ps1

```diff
@@ -13,16 +13,18 @@ if ($IsWindows) {
 }
 $bundle = Join-Path $TestingRoot "bundle"
 $manifestdir = Join-Path $TestingRoot "manifest"
+$commonArgs = @(
+    "--vcpkg-root=$bundle",
+    "--overlay-triplets=$env:VCPKG_ROOT/triplets",
+    "--overlay-triplets=$env:VCPKG_ROOT/triplets/community",
+    "--x-scripts-root=$env:VCPKG_ROOT/scripts"
+)
 
 # Test classic bundle
 New-Item -ItemType Directory -Force $bundle | Out-Null
 New-Item -ItemType File -Force $bundle/.vcpkg-root | Out-Null
 
-$a = Run-Vcpkg z-print-config `
-    --vcpkg-root=$bundle `
-    --overlay-triplets=$env:VCPKG_ROOT/triplets `
-    --overlay-triplets=$env:VCPKG_ROOT/triplets/community `
-    --x-scripts-root=$env:VCPKG_ROOT/scripts
+$a = Run-Vcpkg z-print-config @commonArgs
 $a
```

Comment:
```suggestion
```

Is this the source of the double output you mentioned in the other PR?

Reply:
Ditto other occurances.

Reply:
Kind of. Every time the result of `Run-Vcpkg` is not assigned or not piped to `| Out-Null` the output is doubled.

Reply:
The output is supposed to be put into `$a`. The line that is just `$a` will print the output again.

Reply:
Yeah.

Reply:
Based on this statement:

> Kind of. Every time the result of `Run-Vcpkg` is not assigned or not piped to `| Out-Null` the output is doubled.

With this `$a` the output should be tripled

Reply:
It is only doubled because the output of `Run-Vcpkg` is assigned. The question is how do we solve this?
1. Use `| Out-Null` everywhere where the output is not assigned (a lot of situation, is likely to be forgotten)
2. `Run-Vcpkg` always prints and only returns the output if a switch `-GetOutput` is passed (Only a few occurrences)
3. `Run-Vcpkg` returns the output. The output must be printed when saved inside a variable (like with the `$a`)

I prefer option 2 > 3 > 1.

Reply:
How about:

4. Use `Run-Vcpkg` to mean "I don't want the output" and `Run-VcpkgAndCaptureOutput` to mean "I want the output"?

Effectively this is (2) but without the ambiguity in whether -GetOutput is a switch intended to be passed to vcpkg itself or a parameter processed entirely in powershell world.

Reply:
Sounds good. I will create a PR for that.